stytch/api/users.py

```diff
@@ -16,7 +16,7 @@ def _validate_attributes(self, attributes: Dict[str, str]) -> bool:
 
     def create(
         self,
-        email: str,
+        email: str = None,
```

Comment:
Same comment as on the Ruby PR, do we want to send values for `phone_number` and `email`  in `data` if they're `= None`?